src/browser/browser.ts

```diff
@@ -64,13 +57,35 @@ export class Browser {
     options.height = parseInt(options.height as string, 10) || 500;
     options.timeout = parseInt(options.timeout as string, 10) || 30;
 
-    if (options.width > 3000 || options.width < 10) {
-      options.width = 2500;
+    if (options.width > this.config.maxWidth || options.width < 10) {
+      options.width = this.config.maxWidth;
+    }
+
+    if (options.width < 10) {
```

Comment:
The only reason width could be less than 10 at this stage is if this.config.maxWidth is less than 10 right? Would it make sense to ensure this.config.maxWidth >= 10 earlier?

Reply:
Thanks for noticing. Now it uses the default width/height if width/height is less than 10.